src/protocol.cpp

```diff
@@ -84,6 +85,7 @@ const static std::string allNetMessageTypes[] = {
     NetMsgType::GETCFCHECKPT,
     NetMsgType::CFCHECKPT,
     NetMsgType::WTXIDRELAY,
+    NetMsgType::SENDRECON,
```

Comment:
In this PR it might be clearer to standardize on "reconcile" / "reconciliation" in the codebase, both for simplicity and to reduce confusion, as "recon" is a different word with a different meaning (as I mentioned in an earlier comment).

In any case, it would be better to do it now while introducing the naming, than later on.